redlines/redlines.py

```diff
@@ -133,22 +133,43 @@ def opcodes(self) -> list[tuple[str, int, int, int, int]]:
     def output_markdown(self) -> str:
         """Returns the delta in Markdown format."""
         result = []
-        style = "red"
+        default_style = 'red_green'
+        style = default_style
+
+        elem_attributes = {
+            "ins": { "class": "inserted" },
+            "del": { "class": "deleted" },
+        }
 
         if self.options.get("markdown_style"):
             style = self.options["markdown_style"]
 
         if style == "none":
             md_styles = {"ins": ("ins", "ins"), "del": ("del", "del")}
-        elif "red":
+        elif style == 'red_green':
+            md_styles = {
+                "ins": (
+                    f"span {elem_attributes['ins']['class']} style='color:green;font-weight:700;'", 
+                    'span'
+                ),
+                "del": (
+                    f"span {elem_attributes['del']['class']} style='color:red;font-weight:700;text-decoration:line-through;'",
+                    'span'
+                )
+            }
+        elif style == "red":
             md_styles = {
-                "ins": ('span style="color:red;font-weight:700;"', "span"),
+                "ins": (
+                    f"span {elem_attributes['ins']['class']} style='color:red;font-weight:700;'", 
```

Comment:
Is this a custom element attribute for the <span>? The result will look like `<span inserted style='color:red;font-weight:700;'>text</span>`? I am not familiar with this. Is it valid in HTML? If it is valid, what is the functionality of this element attribute? thank you.

Reply:
Thank you, I just pushed a change to the branch that fixes the initial attribute implementation. It produces the correct syntax now: `<span class='inserted' style='color:red;font-weight:700;'>text</span>`

The attributes are meant to serve potential future implementations of HTML output, and to help with their styling respectively.